snuba/admin/views.py

```diff
@@ -73,6 +75,27 @@ def health() -> Response:
     return Response("OK", 200)
 
 
+@application.route("/migrations/<group>/list")
+def migrations_groups(group: str) -> Response:
```

Comment:
Is this supposed to be run on a per host basis? Like the system queries one where you select the host on which you want to run the command?

Reply:
This should connect to the migrations cluster for which there is only one node as far as I know